src/components/hand-tracking-controls.js

```diff
@@ -204,44 +205,40 @@ module.exports.Component = registerComponent('hand-tracking-controls', {
   detectPinch: (function () {
     var thumbTipPosition = new THREE.Vector3();
     return function () {
+      var el = this.el;
       var frame = this.el.sceneEl.frame;
       var indexTipPosition = this.indexTipPosition;
+      var wristWorldOrientation = this.wristWorldOrientation;
+      var pinchWorldPosition = this.pinchWorldPosition;
       var controller = this.el.components['tracked-controls'] && this.el.components['tracked-controls'].controller;
       var trackedControlsWebXR = this.el.components['tracked-controls-webxr'];
       var referenceSpace = this.referenceSpace || trackedControlsWebXR.system.referenceSpace;
       if (!controller.hand[XRHand.INDEX_PHALANX_TIP] ||
           !controller.hand[XRHand.THUMB_PHALANX_TIP]) { return; }
       var indexTipPose = frame.getJointPose(controller.hand[XRHand.INDEX_PHALANX_TIP], referenceSpace);
       var thumbTipPose = frame.getJointPose(controller.hand[XRHand.THUMB_PHALANX_TIP], referenceSpace);
+      var wristPose = frame.getJointPose(controller.hand[XRHand.WRIST], referenceSpace);
 
-      if (!indexTipPose || !thumbTipPose) { return; }
+      if (!indexTipPose || !thumbTipPose || !wristPose) { return; }
 
       thumbTipPosition.copy(thumbTipPose.transform.position);
+      thumbTipPosition.y += 1.5;
       indexTipPosition.copy(indexTipPose.transform.position);
+      indexTipPosition.y += 1.5;
+      wristWorldOrientation.copy(wristPose.transform.orientation);
```

Comment:
What is the use case for the wrist orientation? Is not wrist position more useful?